server/tournaments/index.ts

```diff
@@ -1699,6 +1727,33 @@ const chatCommands: ChatCommands = {
 					}
 				}
 			}
+		} else if (cmd === 'export') {
+			if (room.lastTournamentData && room.lastTournamentData.type === 'tree') {
+				let tourString = '';
+				let tourData = [];
+				const tourWinner = room.lastTournamentData.rootNode.team;
+				let roundCounter = Math.ceil(Math.log(room.lastTournamentData.playersLength) / Math.log(2));
+				tourData.push(room.lastTournamentData.rootNode);
+				while (tourData.some(node => node.children)) {
+					const tourDataCopy: TournamentDataNode[] = tourData;
```

Comment:
Naming here is kind of a mess, IMO. `Copy` is a smell. this also feels backwards = why not name the new variable below something other than `tourData` (maybe something to do with `round`, that is what this is iterating over, no?) and reassign `tourData` at the bottom of the loop as well? Also, `copyChild` seems like it should just be `child` followed by `grandChild` instead of `copyChild` and `child`?